library/src/main/java/com/felipecsl/gifimageview/library/GifDecoder.java

```diff
@@ -95,6 +95,8 @@
 
   private static final int INITIAL_FRAME_POINTER = -1;
 
+  static final int LOOP_FOREVER = 0;
```

Comment:
I would think that it might be useful to expose this as a public variable so that it could be compared against externally explicitly after someone pulls a value from getLoopCount()